widget/select_entry.go

```diff
@@ -57,3 +58,19 @@ func (e *SelectEntry) SetOptions(options []string) {
 	dropDownButton.SetIcon(theme.MenuDropDownIcon())
 	e.ActionItem = dropDownButton
 }
+
+type cursorableButton struct {
+	Button
+}
+
```

Comment:
Why not implement `Cursorable` in `Button` instead?

Reply:
IMHO, I don't think that's the best thing to do. By default, any non-cursorable widget has `fyne.DefaultCursor`. I see no need of allowing `Button` implement `Cursorable` for the sake of `SelectEntry`. If button implement `Cursorable`, it's cursor will still remain `fyne.DefaultCursor`. Does it really make sense?

Reply:
> Does it really make sense?

IMO yes. The implementation of `Cursorable` means that a component has a specific wish regarding the cursor. Any component that does not implement `Cursorable` simply doesn't care about the cursor.

Reply:
Plus, I see no need to implement a special button just for the sake of the cursor in the `SelectEntry` :).

Reply:
The drop-down button should behave like every other button. That's why it should not be a special button.

Reply:
I have thought this over more and agree with @toaster - all buttons should have the same cursor, so let's just put this on widget.Button :)